src/uct/api/uct.h

```diff
@@ -1969,6 +2000,32 @@ UCT_INLINE_API ucs_status_t uct_ep_atomic_cswap32(uct_ep_h ep, uint32_t compare,
 }
 
 
+/**
+ * @ingroup UCT_AMO
+ * @brief
+ */
+UCT_INLINE_API ucs_status_t uct_ep_atomic_post(uct_ep_h ep, uct_atomic_post_op_t opcode,
+                                               uint64_t value, size_t size,
```

Comment:
maybe define uct_ep_atomic_post32, uct_ep_atomic_post64, uct_ep_atomic_fetch32_nb, uct_ep_atomic_fetch64_nb? instead of checking size
the size can be up to 64 anyway..
@shamisp what do you think?

Reply:
will update after final decision